### PR TITLE
Template update for nf-core/tools version 2.14.1

### PR DESCRIPTION
**Pull request by LilyAnderssonLee**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/metaval/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/metaval _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nf-test test main.nf.test -profile test,docker`).
- [ ] Check for unexpected warnings in debug mode (`nextflow run . -profile debug,test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,58 +5,65 @@
   </picture>
 </h1>
 
-[![GitHub Actions CI Status](https://github.com/genomic-medicine-sweden/tomte/workflows/nf-core%20CI/badge.svg)](https://github.com/genomic-medicine-sweden/meta-val/actions?query=workflow%3A%22nf-core+CI%22)
-[![GitHub Actions Linting Status](https://github.com/genomic-medicine-sweden/tomte/workflows/nf-core%20linting/badge.svg)](https://github.com/genomic-medicine-sweden/meta-val/actions?query=workflow%3A%22nf-core+linting%22)[![AWS CI](https://img.shields.io/badge/CI%20tests-full%20size-FF9900?labelColor=000000&logo=Amazon%20AWS)](https://nf-co.re/metaval/results)[![Cite with Zenodo](http://img.shields.io/badge/DOI-10.5281/zenodo.XXXXXXX-1073c8?labelColor=000000)](https://doi.org/10.5281/zenodo.XXXXXXX)
+[![GitHub Actions CI Status](https://github.com/nf-core/metaval/actions/workflows/ci.yml/badge.svg)](https://github.com/nf-core/metaval/actions/workflows/ci.yml)
```
Comment (sofstam) [inline README.md]: Should this here be `nf-core`?

File: assets/samplesheet.csv
Comment (sofstam) [inline assets/samplesheet.csv:1]: I am confused here, shouldn't here be an example of a samplesheet you have been working with?

File: assets/schema_input.json
Comment (sofstam) [inline assets/schema_input.json:1]: Same here for the schema.

**Review by sofstam — COMMENTED** [review]
Nice! A few comments :)

File: README.md (hunk 1 above)
Comment (LilyAnderssonLee) [reply README.md]: I should skip the update of README.md

File: assets/samplesheet.csv
Comment (LilyAnderssonLee) [reply assets/samplesheet.csv:1]: This one was added by the template. I think I could skip adding it here, right?

File: assets/schema_input.json
Comment (LilyAnderssonLee) [reply assets/schema_input.json:1]: Thanks for pointing out. I didn't check it carefully. I need to keep the one on the dev branch.

File: assets/samplesheet.csv
Comment (sofstam) [reply assets/samplesheet.csv:1]: I think we should use our samplesheet. 
Looking at tomte for example:

https://github.com/genomic-medicine-sweden/tomte/blob/dev/assets/samplesheet.csv

**Comment by sofstam** [comment]
Nice! Let's merge this!

**Pull request merged**